server/src/main/java/org/elasticsearch/plugins/ClusterCoordinationPlugin.java

```diff
@@ -65,7 +66,8 @@ PersistedClusterStateService newPersistedClusterStateService(
             NodeEnvironment nodeEnvironment,
             NamedXContentRegistry xContentRegistry,
             ClusterSettings clusterSettings,
-            LongSupplier relativeTimeMillisSupplier
+            LongSupplier relativeTimeMillisSupplier,
+            ThreadPool threadPool
```

Comment:
Do we need both `relativeTimeMillisSupplier` and `threadPool` since relative time could be obtained from `threadPool.relativeTimeInMillis()` now?

Reply:
++ I also think we just need the `threadPool`.

Reply:
Addressed in https://github.com/elastic/elasticsearch/pull/95206/commits/0862a5d7d375ce6fcc6198e845cd2887e7c88e90